packages/samples/headless-react/src/App.tsx

```diff
@@ -4,6 +4,7 @@ import {BrowserRouter, NavLink, Switch, Route} from 'react-router-dom';
 import {RecommendationPage} from './pages/RecommendationPage';
 import {StandaloneSearchBoxPage} from './pages/StandaloneSearchBoxPage';
 import {SamlPage} from './pages/SamlPage';
+import {DependentFacet} from './pages/DependentFacet';
```

Comment:
Could it be clearer if this component was called `DependentFacetPage`, but also to stay consistent?

At first read I thought `DependentFacet`  was a single Facet component,